src/applications/vaos/utils/selectors.js

```diff
@@ -69,3 +83,14 @@ export function getFacilityPageInfo(state, pageKey) {
       !!formInfo.schema && !!formInfo.schema.properties.vaFacilityMessage,
   };
 }
+
+export function getChosenClinicInfo(state) {
```

Comment:
This is somewhat speculative. I'm assuming we'll store clinics the same way we store facilities, cached according to the data used to fetch them, so we don't have to refetch if someone backs up and changes facilities.